modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1678,7 +1678,7 @@
                 "format": "java",
                 "type": "Object",
                 "annotations": {
-                  "typeId": "org.mule.runtime.core.routing.AggregationStrategy",
+                  "typeId": "org.mule.runtime.core.internal.routing.AggregationStrategy",
```

Comment:
@alegmarra, does this kind of annotation mean that tooling should be able to access this interface to allow users to contribute with a custom strategy?